contracts/duckies/nft/DucklingFamilyGame/DucklingFamilyGameV1.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import '@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol';
+import '@openzeppelin/contracts-upgradeable/utils/StringsUpgradeable.sol';
+import '@openzeppelin/contracts-upgradeable/utils/cryptography/ECDSAUpgradeable.sol';
+
+import '../../../interfaces/IVoucher.sol';
+import '../RandomUpgradeable.sol';
+import '../Gene.sol';
+
+contract DucklingFamily is IVoucher, RandomUpgradeable {
```

Comment:
Upgradability through Random parent is strange

Reply:
DuckyFamily is not utilizing all upgradeability contracts yet. `RandomUpgradeable` is called so because it is meant to be used for upgradeability.